dashboard/db/migrate/20160406190753_create_join_table_plc_learning_modules_plc_tasks.rb

```diff
@@ -0,0 +1,16 @@
+class CreateJoinTablePlcLearningModulesPlcTasks < ActiveRecord::Migration
+  def up
+    create_join_table :plc_learning_modules, :plc_tasks do |t|
+      t.index :plc_learning_module_id
+      t.index :plc_task_id
+    end
+    execute 'insert into plc_learning_modules_tasks (plc_learning_module_id, plc_task_id) select plc_learning_module_id, id from plc_tasks;'
```

Comment:
You can combine lines 3 to 7 with one "create table as " statement if desired. Not sure if it actually is desired